### PR TITLE
Revert "AIT-9184: update gotraceui to v0.3.0."

### PR DESCRIPTION
**Pull request by darccio**
Reverts DataDog/dd-trace-go#2443 It was merged accidentally.

### CODE REVIEW

**Review by nsrip-dd — APPROVED** [review]
Looks like you'll need tracing-go approval as well due to code owners

**Review by ajgajg1134 — APPROVED** [review]

**Pull request merged**
